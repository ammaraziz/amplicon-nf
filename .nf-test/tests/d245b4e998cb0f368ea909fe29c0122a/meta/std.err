[0;31mThe following invalid input values have been detected:

* Missing required parameter(s): input
[0m
 -- Check script '/Users/sam/Library/CloudStorage/OneDrive-SharedLibraries-onedrive/bioinformatics/artic-repos/artic-network-fieldbioinformatics-nf/./workflows/../subworkflows/local/utils_nfcore_amplicon-nf_pipeline/../../nf-core/utils_nfschema_plugin/main.nf' at line: 39 or see '/Users/sam/Library/CloudStorage/OneDrive-SharedLibraries-onedrive/bioinformatics/artic-repos/artic-network-fieldbioinformatics-nf/.nf-test/tests/d245b4e998cb0f368ea909fe29c0122a/meta/nextflow.log' file for more details
