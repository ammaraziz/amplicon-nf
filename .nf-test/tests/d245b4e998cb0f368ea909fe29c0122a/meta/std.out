ERROR ~ Validation of pipeline parameters failed!

 -- Check '/Users/sam/Library/CloudStorage/OneDrive-SharedLibraries-onedrive/bioinformatics/artic-repos/artic-network-fieldbioinformatics-nf/.nf-test/tests/d245b4e998cb0f368ea909fe29c0122a/meta/nextflow.log' file for details
